faker/providers/misc/__init__.py

```diff
@@ -70,6 +71,13 @@ def language_code(cls):
         return cls.random_element(cls.language_codes)
 
     @classmethod
+    def uuid(cls):
```

Comment:
I've only ever used `uuid4` so I think that is sufficient. Do I need to add any tests/docs?